external-docs/docs/guides/errors/build-command-errors/multiple-observations-columns.md

```diff
@@ -1,4 +1,4 @@
-# Error - multiple observations columns defined (Deprecated)
+++++++++++++++++++++++# Error - multiple observations columns defined (Deprecated)
```

Comment:
?

Reply:
This was a reminder to myself that this may need to be amended once the pivoted shape feature is released

Reply:
Possible cat on keyboard incident